pkg/cmd/crud/crud.go

```diff
@@ -0,0 +1,76 @@
+package crud
+
+import (
+	"fmt"
+	"os"
+	"text/template"
+
+	"github.com/aerogear/charmil/core/factory"
+	tmpl "github.com/aerogear/charmil/pkg/template"
+	"github.com/spf13/cobra"
+)
+
+type FlagVariables struct {
+	path string
+
+	Singular string
+	Plural   string
+}
```

Comment:
shall we use this github.com/manifoldco/promptui? Instead of flags. To make it cooler!
whatever feels good. Need suggestion here @wtrocki

Reply:
Flags and interactive mode can be supported.
Flags are good start

Reply:
> shall we use this github.com/manifoldco/promptui

I had a look at its readme just now, looks really cool.